<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>usb_gen_v2_qtClass</class>
 <widget class="QMainWindow" name="usb_gen_v2_qtClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>154</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>459</width>
    <height>154</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>459</width>
    <height>154</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Генератор USB (25-6000 МГц)</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/GEN_USB/Resources/ico.ico</normaloff>:/GEN_USB/Resources/ico.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>413</width>
     <height>86</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>500</width>
     <height>200</height>
    </size>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>461</width>
      <height>141</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="documentMode">
     <bool>true</bool>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Управление</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>40</y>
        <width>95</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Сохранить</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>10</y>
        <width>95</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Включить</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>5</y>
        <width>115</width>
        <height>61</height>
       </rect>
      </property>
      <property name="title">
       <string>Частота</string>
      </property>
      <widget class="QDoubleSpinBox" name="doubleSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>22</y>
         <width>80</width>
         <height>22</height>
        </rect>
       </property>
       <property name="accelerated">
        <bool>true</bool>
       </property>
       <property name="decimals">
        <number>5</number>
       </property>
       <property name="minimum">
        <double>25.000000000000000</double>
       </property>
       <property name="maximum">
        <double>6000.000000000000000</double>
       </property>
       <property name="value">
        <double>2000.000000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>22</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>МГц</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>125</x>
        <y>5</y>
        <width>91</width>
        <height>61</height>
       </rect>
      </property>
      <property name="title">
       <string>Ослабление</string>
      </property>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>22</y>
         <width>50</width>
         <height>22</height>
        </rect>
       </property>
       <property name="accelerated">
        <bool>true</bool>
       </property>
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="minimum">
        <double>-31.500000000000000</double>
       </property>
       <property name="maximum">
        <double>0.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.500000000000000</double>
       </property>
       <property name="value">
        <double>-31.500000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>65</x>
         <y>22</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string> дБ</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>5</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="title">
       <string>Режим</string>
      </property>
      <widget class="QRadioButton" name="radioButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>15</y>
         <width>41</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>ГСС</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>35</y>
         <width>41</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>ГКЧ</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_3">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>15</y>
         <width>51</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>ППРЧ</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>70</y>
        <width>451</width>
        <height>41</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_7">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>41</x>
         <y>10</y>
         <width>55</width>
         <height>22</height>
        </rect>
       </property>
       <property name="accelerated">
        <bool>true</bool>
       </property>
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="minimum">
        <double>25.000000000000000</double>
       </property>
       <property name="maximum">
        <double>6000.000000000000000</double>
       </property>
       <property name="value">
        <double>2000.000000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>10</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>МГц</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>3</x>
         <y>10</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Начало</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>10</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Конец</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>225</x>
         <y>10</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>МГц</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_8">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>165</x>
         <y>10</y>
         <width>55</width>
         <height>22</height>
        </rect>
       </property>
       <property name="accelerated">
        <bool>true</bool>
       </property>
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="minimum">
        <double>25.000000000000000</double>
       </property>
       <property name="maximum">
        <double>6000.000000000000000</double>
       </property>
       <property name="value">
        <double>2100.000000000000000</double>
       </property>
      </widget>
      <widget class="Line" name="line">
       <property name="geometry">
        <rect>
         <x>115</x>
         <y>10</y>
         <width>20</width>
         <height>21</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="Line" name="line_2">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>10</y>
         <width>20</width>
         <height>21</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>355</x>
         <y>10</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Время</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_2">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>10</y>
         <width>40</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>435</x>
         <y>10</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>мс</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>325</x>
         <y>10</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>МГц</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_9">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>10</y>
         <width>43</width>
         <height>22</height>
        </rect>
       </property>
       <property name="accelerated">
        <bool>true</bool>
       </property>
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="minimum">
        <double>0.100000000000000</double>
       </property>
       <property name="maximum">
        <double>9.990000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>0.500000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_18">
       <property name="geometry">
        <rect>
         <x>255</x>
         <y>10</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Шаг</string>
       </property>
      </widget>
      <widget class="Line" name="line_3">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>10</y>
         <width>20</width>
         <height>21</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Настройки</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>10</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Обновление FW</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>XTAL=</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>R=</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_3">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="accelerated">
       <bool>true</bool>
      </property>
      <property name="decimals">
       <number>5</number>
      </property>
      <property name="minimum">
       <double>25.000000000000000</double>
      </property>
      <property name="maximum">
       <double>350.000000000000000</double>
      </property>
      <property name="value">
       <double>350.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>10</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Version FW :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>40</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Version SW : 1.5</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>16383</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>10</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>МГц</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Nint =</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Nfract =</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>70</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>K =</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action">
   <property name="text">
    <string>АЧХ</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Об авторе</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
